roles/openshift-applier/tasks/process-template.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: "Check if params file exists (if applicable)"
+  stat:
+    path: "{{ tmp_inv_dir }}{{ params }}"
+  ignore_errors: yes
+  register: params_result
+  when:
+  - params|trim != ''
+
+- name: "Fail if params file doesn't exist (if applicable)"
+  fail:
+    msg: "{{ params }} - params file doesn't exist."
+  when:
+  - params|trim != ''
+  - params_result.stat.exists == False
+
+- name: "Determine location for the 'template'"
+  vars:
+    path: "{{ template }}"
+  import_tasks: check-file-location.yml
+
+- name: "Set 'template' values"
+  set_fact:
+    template: "{{ file_path }}"
+    template_f_option: "{{ option_f }}"
+
+- name: "Process template with params from vars"
```

Comment:
BTW: unless I'm missing something, the logic for these 2 tasks means that the template is *not* processed if both `params` and `params_from_vars` is set. This is the opposite of intended behavior I believe. Basically, we'd want the `params_from_vars` to "override" the params in the file if they are set. Hence, this is another reason for not splitting into two files but rather combine the two.

Reply:
You are correct. Because dealing with collisions and remediation of the two different locations is not in scope for this PR. It will be addressed separately in a different issue

Reply:
hmm, no - let's not ping/pong on this. I'll ping you offline to figure out best resolution to this.